.gitmodules

```diff
@@ -836,6 +836,9 @@
 [submodule "vendor/grammars/razor-plus"]
 	path = vendor/grammars/razor-plus
 	url = https://github.com/austincummings/razor-plus
+[submodule "vendor/grammars/rescript-vscode"]
+	path = vendor/grammars/rescript-vscode
+	url = https://github.com/rescript-lang/rescript-vscode
```

Comment:
This grammar doesn't have a license so can't be added to Linguist until such time as it does, and it would need to be one of these:

https://github.com/github/linguist/blob/c362ab8b3f4f66cbfa7568cf8cb20f7a560de928/vendor/licenses/config.yml#L3-L14

Reply:
The repository was updated with a MIT license.